### PR TITLE
Add Fugu landing page

### PR DESCRIPTION
**Pull request by tomayac**
Changes proposed in this pull request:

- Add Fugu landing page.

### CODE REVIEW

File: site/en/blog/fugu-showcase/index.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ title: Project Fugu API showcase
 subhead: >
   The Project Fugu API Showcase is a collection of apps that make use of APIs that were conceived in
   the context of Project Fugu.
+description: The Project Fugu API Showcase is a collection of apps that make use of APIs that were conceived in the context of Project Fugu.
```
Comment (jpmedley) [inline site/en/blog/fugu-showcase/index.md:7]:
```suggestion
description: The Project Fugu API Showcase is a collection of apps that use APIs created in the context of Project Fugu.
```

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: site/en/blog/fugu-showcase/index.md (hunk 1 above)
Comment (tomayac) [reply site/en/blog/fugu-showcase/index.md:7]: This PR is far from being in a reviewable state, but thanks, I’ll happily take this drive-by review.

Comment (jpmedley) [reply site/en/blog/fugu-showcase/index.md]: Sorry about that. It looked like it was ready when I came in.

**Comment by tomayac** [comment]
This is now ready for review.

**Review by petele — APPROVED** [review]

**Comment by tomayac** [comment]
@rachelandrew, this is now ready for Tech Writer review. @petele has reviewed for content.

**Comment by rachelandrew** [comment]
@tomayac you have an additional resources box bottom right that is currently empty. This is looking nice though!

I'd really like to get the remaining content linked here moved off web.dev. I'm going to get Aaron to look at it as soon as he has a signed CLA to allow him to write to GitHub.

**Comment by tomayac** [comment]
> [Y]ou have an additional resources box bottom right that is currently empty. This is looking nice though!

This looks right to me. It's the same as the Workbox one, just with two boxes instead of more. Or what do you mean?

Screenshot 2022-09-22 at 11 10 34

> I'd really like to get the remaining content linked here moved off web.dev. I'm going to get Aaron to look at it as soon as he has a signed CLA to allow him to write to GitHub.

Nice, thanks. Looking forward to having everything on one site. Less clutter and confusion.

File: site/en/capabilities/index.njk [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+      <h2 class="type--h4">
+        <a class="surface display-inline-flex color-text" href="{{item.url}}">
+          {{ item.title }}
+        </a>
+      </h2>
+      <p class="flex-1">{{ item.description }}</p>
+    </div>
+  {% endfor %}
+  {{ landingDeco(
+    'Additional resources',
```
Comment (rachelandrew) [inline site/en/capabilities/index.njk:201]: I think you need to place this heading first, there is CSS that will place is visually last.

**Comment by tomayac** [comment]
@rachelandrew I think we're good now. I'll fix the Workbox page as well, see https://github.com/GoogleChrome/developer.chrome.com/pull/3799.

**Review by rachelandrew — APPROVED** [review]

**Pull request merged**